doc: add notice about custom configuration for CF-pages (#5453)
sveltejs/kit: packages/adapter-cloudflare/README.md

Functions contained in the `/functions` directory at the project's root will _not_ be included in the deployment, which is compiled to a [single `_worker.js` file](https://developers.cloudflare.com/pages/platform/functions/#advanced-mode). Functions should be implemented as [endpoints](https://kit.svelte.dev/docs/routing#endpoints) in your SvelteKit app.

> If you want to use `_headers` or `_redirects` custom [config files](https://developers.cloudflare.com/pages/platform/headers) to modify cloudflare behaviour you should put those configs to `/static` folder of yours SvelteKit project, NOT to the root of SvelteKit project.

Reviewer: ```suggestion
If you want to use `_headers` or `_redirects` custom [config files](https://developers.cloudflare.com/pages/platform/headers) to modify cloudflare behaviour you should put those configs in the `/static` folder of your SvelteKit project.
```
